app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/response/RegResponse.java

```diff
@@ -0,0 +1,9 @@
+package com.greenfox.peridot.peridot_coz_android.model.response;
+
+/**
+ * Created by mozgaanna on 30/01/17.
+ */
+
+public class RegResponse extends Response{
+    UserResponse user;
+}
```

Comment:
@mozgaanna if this is the same in the specs (my bad), then don't create the same object, just use the same for both eg. `LoginAndRegResponse`